ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/clientlibs/editor/css/panelcontainer.css

```diff
@@ -0,0 +1,23 @@
+.cmp-adaptiveform-panelcontainer__repeatable label.coral-Form-fieldlabel {
+    display: inline-block;
+    margin-top: 8px;
+    margin-left: 9px;
+}
+
+.cmp-adaptiveform-panelcontainer__repeatable coral-switch.coral-Form-field {
+    display: inline-block;
+    margin-top: 10px;
+    margin-left: 4px;
+    float: left;
+}
+
+.cmp-adaptiveform-panelcontainer__minOccur,
+.cmp-adaptiveform-panelcontainer__maxOccur {
+    margin-left: 25px !important;
```

Comment:
any reason for important annotations?.. this can make dialog customizations/styling hard.

Reply:
this is dialog, and needs important, or we would need to make it hierarchical selector.